src/ripple/app/misc/Validations.cpp

```diff
@@ -387,6 +387,39 @@ class ValidationsImp : public Validations
         return ret;
     }
 
+    std::uint32_t getValidationTime (uint256 const& hash, int minValidations)
```

Comment:
This PR is already marked as passed, and this suggestion may be overly nitpicky, but I really prefer the pattern where we return a pair with a success flag, rather than a magic number (even if it is 0) on failure.